### DISCUSSION TITLE
Secret key

### DISCUSSION THREAD

**Discussion by wtfglock** [ente-io/ente#3412] [Q&A]
Dear Ente,

I'm trying to set up an ente family account and updating my subscription 200 gigs, but the app is asking me for my secret key. It's not my password, neither is it my recovery code. 

Could you please let me know what a secret key is and how I can find if of reser it? 

Thanks in advance  

Kind regards,
Jesse

**Comment by ua741** [comment]
Hey, Can you please share screenshot for the screen after hiding any personal details?
